.gitignore

```diff
@@ -0,0 +1,38 @@
+*.pyc
+*.swp
+.DS_Store
+/ve/
+/*.egg-info/
+/.coverage
+/coverage-html/
+/coverage.xml
+/node_modules/
+/bower_components/
+/nosetests.xml
+/pyflakes.log
+/pep8.log
+/htmlcov/
+/_trial_temp/
+/config/*
+/logs/*
+/tmp/*
+requirements.pip.md5
+.project
+.pydevproject
+*.komodoproject
+*.sublime-project
+*.sublime-workspace
+distribute-0.6.10.tar.gz
+.DS_Store
+*.db
+/go/production_settings.py
+/setup_env/build/
+/.tox/
+/test_results.xml
+/pep8.txt
+/docs/_build/
+/.codeintel/
+mochacov.lcov
+/coverage/
+/go/static
```

Comment:
This .gitignore seems to have been copied across from Vumi Go. Perhaps we should copy over one that doesn't have quite so many Vumi Go specific things in it? Like maybe the one from go-contacts-api -- https://github.com/praekelt/go-contacts-api/blob/develop/.gitignore ?